app/controllers/games_controller.rb

```diff
@@ -1,4 +1,7 @@
 class GamesController < ApplicationController
+  before_action :authenticate_user!, only: [
+    :new, :create, :edit, :update, :destroy
+    ]
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.